src/server/game/Server/WorldSocket.cpp

```diff
@@ -119,8 +119,9 @@ bool WorldSocket::IsClosed(void) const
     return closing_;
 }
 
-void WorldSocket::CloseSocket(void)
+void WorldSocket::CloseSocket(std::string const& reason)
 {
+    sLog->outDebug(LOG_FILTER_CLOSE_SOCKET, "Socket closed because of: %s", reason.c_str());
```

Comment:
Check if a reason was given first

Reply:
+

Reply:
@Deku @Viste thanks, please check it now